### DISCUSSION TITLE
List of supported format

### DISCUSSION THREAD

**Discussion by pputnik** [SubtitleEdit/subtitleedit#6993] [General]
Hi there, I know there's a list of formats directly in the app but sometimes you need the list to send it to someone else to select the common type of files you're going to work with.

I haven't found it in doc or repo, so I had to do it myself, grep-ing code. It'd be nice to have it in the app/doc in future versions (when the list below becomes outdated). As of Subtitle Edit 3.6.13, the list is as follows:

.1HD ARIB
.890 Cavena 890
.C Unknown 20
.STL DVD Studio Pro
.STL DVD Studio Pro with one space
.STL DVD Studio Pro with one space/semicolon
.STL DVD Studio Pro with space
.STL DVD Studio Pro with space
.WSB WSB
.aqt AQTitle
.asc Timeline ascii
.asc Timeline footage ascii
.asc Unknown 30
.ass Advanced Sub Station Alpha
.atpj JSON Archtime project
.ats ATS ISS
.aya Ayato
.bcc Bilibili json
.cac Caption Assistant
.cap CapMaker Plus
.cap Cheetah Caption
.cap Cheetah Caption Old
.cap Lambda Cap
.cap NCI Caption
.caption PList Caption xml
.chk CHK
.cin Caption Inc
.cip Unknown 16
.cmft CMAF Text
.crk Caraoke Xml
.csv Csv
.csv Csv Nuendo
.csv Csv2
.csv Csv3
.csv Csv4
.csv Csv5
.csv Edius Marker List 2 Frames
.csv Edius Marker List 2 Ms
.csv Edius Marker List 3 Frames
.csv Edius Marker List 3 Ms
.ctm Unknown 90
.detx Cappella
.dfxp DFXP Basic
.dfxp Netflix Timed Text
.dfxp Timed Text draft 2006-04 Ooyala
.dost DOST
.dtc Drtic
.dv DV Subtitle
.edl EDL
.elr ELRStudio Closed Caption
.eztxml EZT XML
.fcpxml Final Cut Pro X Chapter Marker
.fcpxml Final Cut Pro X Xml
.fcpxml Final Cut Pro Xml 1.3
.fcpxml Final Cut Pro Xml 1.4
.fcpxml Final Cut Pro Xml 1.4 Text
.fcpxml Final Cut Pro Xml 1.5
.fcpxml Final Cut Pro Xml Name
.fcpxml Final Cut Xml Gap
.fdx Final Draft Template 2
.flc NCI Timed Roll Up Captions
.fpc PAC Unicode (UniPac)
.fte FTE
.gst Gremots json
.htm dl dd span
.html Html javascript sami array
.html Kanopy Html
.html SE image HTML index
.html SMIL Timesheet
.html Unknown 105
.html Unknown 9
.idx VobSub index file
.imtpro MediaTransData
.inqscr InqScribe 1.1
.ismt HBO GO
.itt iTunes Timed Text
.json AWS Transcribe
.json Google Play json
.json HoliStar Json
.json JSON
.json JSON Aeneas
.json JSON TED
.json JSON Type 10
.json JSON Type 11
.json JSON Type 12
.json JSON Type 13
.json JSON Type 14
.json JSON Type 15
.json JSON Type 16
.json JSON Type 17
.json JSON Type 18
.json JSON Type 19
.json JSON Type 2
.json JSON Type 20
.json JSON Type 3
.json JSON Type 4
.json JSON Type 5
.json JSON Type 6
.json JSON Type 7
.json JSON Type 8
.json JSON Type 8b
.json JSON Type 9
.json JSON Type Only load 1
.json JSON Type Only load 2
.json JSON Type Only load 3
.json JSON Type Only load 4
.json SubUrbia
.json Transcriptive Json
.json TwentyThree json
.json TwentyThree json embed
.json TwentyThree json embed srt
.jss JACOsub
.lrc LRC Lyrics
.lrc LRC Lyrics, no end time
.m2ts Blu-ray TextST
.mcc MacCaption 1.0
.mvt Magic Video Titler
.mvt Timeline mvt
.pac PAC (Screen Electronics)
.pjs Phoenix Subtitle
.pns PNS
.psl Projection Subtitle List
.pst Unknown 46
.rt RealTime
.rtf F4 Rich Text Format
.rtf Image Logic Autocaption
.rtf RTF 1
.rtf RTF 2
.rtf Unknown 101
.rtf Unknown 21
.rtf Unknown 23
.rtf Unknown 24
.rtf Unknown 37
.rtf Unknown 39
.rtf Unknown 45
.rtf Unknown 55
.rtf Unknown 58
.rtf Unknown 83
.rtf Unknown 98
.rtf Unknown 99
.sbv YouTube sbv
.scc Scenarist Closed Captions
.scc Scenarist Closed Captions Drop Frame
.sif Swift Interchange File V2
.smi SAMI
.smi SAMI AVDicPlayer
.smi SAMI YouTube
.smi SAMI modern
.son SON
.spt spt
.sptx sptx
.srt SubRip
.ssa Sub Station Alpha
.sst Sonic Scenarist Bitmaps
.stl Avid STL
.stl EBU STL
.stl Spruce Subtitle File
.stl Spruce Subtitle With Space
.stp Belle Nuit Subtitler
.sub DVDSubtitle
.sub IAI subtitle
.sub MicroDVD
.sub SoftNi colon sub
.sub SoftNi sub
.sub Sony DVDArchitect
.sub Sony DVDArchitect Explicit duration
.sub Sony DVDArchitect Tabs
.sub Sony DVDArchitect w. line#
.sub SubViewer 1.0
.sub SubViewer 2.0
.sub TSB4
.sub Unknown 11
.sub Unknown 25
.sub Unknown 4
.subtitle Tmpeg Encoder Text
.ted TED Caption 2.0
.tek Smart Titler
.titl Unknown 6
.tmm Play Captions FreeEditor
.tmx Translation Memory xml
.trs Transcriber Xml
.tts TurboTitler
.ttxt GPAC TTXT
.tx3g tx3g
.txt Adobe Encore
.txt Adobe Encore (line#/tabs/n)
.txt Adobe Encore (line/tabs)
.txt Adobe Encore (tabs)
.txt Adobe Encore NTSC
.txt Adobe Encore w. line#
.txt Audacity labels
.txt Avid Caption
.txt Avid Caption Drop Frame
.txt Avid DVD
.txt Avid Loc Markers
.txt DVD Subtitle System
.txt DigiBeta
.txt ELR Studio print
.txt F4 Text
.txt FAB Subtitler
.txt Footage
.txt Karaoke CDG Creator text
.txt MPlayer2
.txt MacSub
.txt Midway Inscriber CG-X
.txt OGM Chapters
.txt Oresme
.txt Otter AI transcription
.txt PE2
.txt Pinnacle Impression
.txt QubeMasterPro Import
.txt QuickTime text
.txt RX marker
.txt SatBox png
.txt Scenarist
.txt Sony DVDArchitect line/dur/length
.txt Sony DVDArchitect line/duration
.txt Speechmatics
.txt Structured titles
.txt Swift text
.txt Swift text line#
.txt Swift text line# +dur
.txt TMPlayer
.txt Time Codes Only 1
.txt Time Codes Only 2
.txt Time Codes Only 3
.txt TitleExchange Pro
.txt Titra
.txt Ulead subtitle format
.txt Unknown 1
.txt Unknown 10
.txt Unknown 102
.txt Unknown 103
.txt Unknown 104
.txt Unknown 106
.txt Unknown 107
.txt Unknown 12
.txt Unknown 17
.txt Unknown 18
.txt Unknown 2
.txt Unknown 22
.txt Unknown 26
.txt Unknown 27
.txt Unknown 29
.txt Unknown 3
.txt Unknown 31
.txt Unknown 32
.txt Unknown 33
.txt Unknown 34
.txt Unknown 35
.txt Unknown 36
.txt Unknown 38
.txt Unknown 40
.txt Unknown 41
.txt Unknown 42
.txt Unknown 44
.txt Unknown 47
.txt Unknown 48
.txt Unknown 49
.txt Unknown 50
.txt Unknown 51
.txt Unknown 52
.txt Unknown 53
.txt Unknown 54
.txt Unknown 56
.txt Unknown 57
.txt Unknown 59
.txt Unknown 60
.txt Unknown 61
.txt Unknown 62
.txt Unknown 63
.txt Unknown 64
.txt Unknown 65
.txt Unknown 66
.txt Unknown 68
.txt Unknown 69
.txt Unknown 7
.txt Unknown 70
.txt Unknown 71
.txt Unknown 72
.txt Unknown 73
.txt Unknown 74
.txt Unknown 75
.txt Unknown 77
.txt Unknown 79
.txt Unknown 8
.txt Unknown 80
.txt Unknown 81
.txt Unknown 84
.txt Unknown 85
.txt Unknown 86
.txt Unknown 87
.txt Unknown 88
.txt Unknown 89
.txt Unknown 93
.txt Unknown 94
.txt Unknown 95
.txt Unknown 96
.txt Unknown 97
.txt Whisper Raw
.txt YouTube Transcript
.txt YouTube Transcript one line
.txt nVivo transcript
.ult Ultech 1.30 Caption
.usf Universal Subtitle Format
.utx UTX
.utx UTX (frames)
.vid Unknown 100
.vid Vid Icelandic
.vtt WebVTT
.vtt WebVTT File with#
.w32 Wincaps W32
.xif XIF
.xml ABC iView
.xml Adobe After Effects ft-MarkerExporter
.xml Adobe Premiere PrProj Xml
.xml BDN Xml
.xml Captionate
.xml Captionate MS
.xml Combined xml
.xml D-Cinema SMPTE 2007
.xml D-Cinema SMPTE 2010
.xml D-Cinema SMPTE 2014
.xml D-Cinema interop
.xml EEG 708
.xml ESUB-XF
.xml Edius 4 Frames
.xml Edius 4 Ms
.xml F4 Xml
.xml FLVCoreCuePoints
.xml Film Edit xml
.xml Final Cut Pro Image
.xml Final Cut Pro Test Xml
.xml Final Cut Pro Test2 Xml
.xml Final Cut Pro Xml
.xml Flash Xml
.xml MS Office Workbook
.xml Netflix IMSC 1.1 Japanese
.xml Ninsight Xml
.xml NkhCuePoints
.xml OpenDVT
.xml Oresme Docx document
.xml Rdf1
.xml Rhozet Harmonic
.xml Rhozet Harmonic Image
.xml SMPTE-TT 2052
.xml SPU Image
.xml Subtitle Editor Project
.xml Timed Text 1.0
.xml Timed Text Base64 Image
.xml Timed Text Image
.xml Timed Text draft 2006-04
.xml Timed Text draft 2006-04 CDATA
.xml Timed Text draft 2006-10
.xml UT Subtitle xml
.xml Unknown 13
.xml Unknown 14
.xml Unknown 15
.xml Unknown 19
.xml Unknown 28
.xml Unknown 43
.xml Unknown 5
.xml Unknown 67
.xml Unknown 76
.xml Unknown 78
.xml Unknown 82
.xml Unknown 91
.xml Unknown 92
.xml Vocapia Split
.xml Xml
.xml Xml 2
.xml YouTube Annotations
.xmp XMP
.xsubtitle TMPGEnc AW5
.xsubtitle TMPGEnc VME
.zeg Zero G

**Comment by niksedk** [comment]
You can also do a `SubtitleEdit /help formats` in command line - 335 subtitle formats :o

**Comment by z2526** [comment]
Hey. Why can't I find the following formats in Subtitle Edit 4.0.6? Only .cap Lambda Cap is there.

.cap Cheetah Caption
.cap Cheetah Caption Old

**Comment by niksedk** [comment]
What are you trying to do?

For import, just use File - Open...
For export, use File - Export...


````
C:\Tools\subtitleedit>SubtitleEdit /help formats

Subtitle Edit 4.0.6 - Batch converter

- Supported formats (input/output):
    AdvancedSubStationAlpha
    SubRip
    ABCiView
    AdobeAfterEffectsft-MarkerExporter
    AdobeEncore
    AdobeEncore(line#/tabs/n)
    AdobeEncore(tabs)
    AdobeEncorew.line#
    AdobeEncoreNTSC
    AQTitle
    AvidCaption
    AvidCaptionDropFrame
    AvidDVD
    AvidLocMarkers
    AWSTranscribe
    BelleNuitSubtitler
    Bilibilijson
    Cappella
    CaptionAssistant
    Captionate
    CaptionateMS
    CaraokeXml
    Csv
    Csv2
    Csv3
    Csv4
    Csv5
    CsvNuendo
    D-Cinemainterop
    D-CinemaSMPTE2007
    D-CinemaSMPTE2010
    D-CinemaSMPTE2014
    DFXPBasic
    DigiBeta
    Drtic
    DVDStudioPro
    DVDStudioProwithonespace
    DVDStudioProwithonespace/semicolon
    DVDStudioProwithspace
    DVDSubtitle
    DVDSubtitleSystem
    DVSubtitle
    EBUSTL
    Edius4Frames
    Edius4Ms
    EdiusMarkerList2Frames
    EdiusMarkerList2Ms
    EdiusMarkerList3Frames
    EdiusMarkerList3Ms
    EDL
    EEG708
    ELRStudioprint
    ESUB-XF
    F4Text
    EZTXML
    F4RichTextFormat
    F4Xml
    FABSubtitler
    FilmEditxml
    FinalCutProXml
    FinalCutProXXml
    FinalCutXmlGap
    FinalCutProXmlName
    FinalCutProXChapterMarker
    FinalCutProXml1.3
    FinalCutProXml1.4
    FinalCutProXml1.4Text
    FinalCutProXml1.5
    FinalCutProXml1.6
    FinalCutProXml1.7
    FinalCutProXml1.8
    FinalCutProXml1.9
    FinalCutProXml1.10
    FinalCutProXml1.11
    FinalCutProTestXml
    FinalCutProTest2Xml
    FlashXml
    FLVCoreCuePoints
    Footage
    GooglePlayjson
    GPACTTXT
    Gremotsjson
    HoliStarJson
    ImageLogicAutocaption
    InqScribe1.1
    ATSISS
    iTunesTimedText
    JACOsub
    JSONAeneas
    JSONTED
    JSON
    JSONType2
    JSONType3
    JSONType4
    JSONType5
    JSONType6
    JSONType7
    JSONType8
    JSONType8b
    JSONType9
    JSONType10
    JSONType11
    JSONType12
    JSONType13
    JSONType14
    JSONType15
    JSONType16
    JSONType17
    JSONType18
    JSONType19
    JSONType20
    JSONType21
    JSONType22
    JSONType23
    KanopyHtml
    LambdaCap
    LRCLyrics
    LRCLyricsms
    LRCLyrics,noendtime
    MacSub
    MagicVideoTitler
    MediaTransData
    MicroDVD
    MidwayInscriberCG-X
    MPlayer2
    MSOfficeWorkbook
    NCITimedRollUpCaptions
    NetflixIMSC1.1Japanese
    NetflixTimedText
    NinsightXml
    nVivotranscript
    OGMChapters
    OpenDVT
    Oresme
    OresmeDocxdocument
    OtterAItranscription
    PE2
    PhoenixSubtitle
    PinnacleImpression
    PListCaptionxml
    ProjectionSubtitleList
    QubeMasterProImport
    QuickTimetext
    RealTime
    RhozetHarmonic
    RTF1
    RTF2
    RXmarker
    SAMI
    SAMIAVDicPlayer
    SAMImodern
    SAMIYouTube
    Scenarist
    ScenaristClosedCaptions
    ScenaristClosedCaptionsDropFrame
    SmartTitler
    SMILTimesheet
    SMPTE-TT2052
    SoftNisub
    SoftNicolonsub
    SonyDVDArchitect
    SonyDVDArchitectExplicitduration
    SonyDVDArchitectline/duration
    SonyDVDArchitectline/dur/length
    SonyDVDArchitectTabs
    SonyDVDArchitectw.line#
    Speechmatics
    SpruceSubtitleFile
    SpruceSubtitleWithSpace
    Structuredtitles
    SubStationAlpha
    SubtitleEditorProject
    SubUrbia
    SubViewer1.0
    SubViewer2.0
    SwiftInterchangeFileV2
    Swifttext
    Swifttextline#
    Swifttextline#+dur
    Xml
    Xml2
    TimedText1.0
    TimedTextdraft2006-04
    TimedTextdraft2006-04CDATA
    TimedTextdraft2006-04Ooyala
    TimedTextdraft2006-10
    TimedTextIMSC1.1
    TitleExchangePro
    Titra
    TmpegEncoderText
    TMPGEncAW5
    TMPGEncVME
    TMPlayer
    TranscriberXml
    TranslationMemoryxml
    Tsv1
    Tsv2
    TurboTitler
    TwentyThreejson
    TwentyThreejsonembed
    TwentyThreejsonembedsrt
    UniversalSubtitleFormat
    UTSubtitlexml
    UTX
    UTX(frames)
    Uleadsubtitleformat
    VideoIndexerTranscript
    VocapiaSplit
    WebVTT
    WebVTTFilewith#
    WhisperRaw
    WhisperRaw2
    XIF
    XMP
    YouTubeAnnotations
    YouTubesbv
    YouTubeTranscript
    YouTubeTranscriptoneline
    ZeroG
    Unknown1
    Unknown2
    Unknown3
    Unknown4
    Unknown5
    Unknown6
    Unknown7
    Unknown8
    Unknown9
    Unknown10
    Unknown11
    Unknown12
    Unknown13
    Unknown14
    Unknown15
    Unknown16
    Unknown17
    Unknown18
    Unknown19
    Unknown20
    Unknown21
    Unknown22
    Unknown23
    Unknown24
    Unknown25
    Unknown26
    Unknown27
    Unknown28
    Unknown29
    Unknown30
    Unknown31
    Unknown32
    Unknown33
    Unknown34
    Unknown35
    Unknown36
    Unknown37
    Unknown38
    Unknown39
    Unknown40
    Unknown41
    Unknown42
    Unknown43
    Unknown44
    Unknown45
    Unknown46
    Unknown47
    Unknown48
    Unknown49
    Unknown50
    Unknown51
    Unknown52
    Unknown53
    Unknown54
    Unknown55
    Unknown56
    Unknown57
    Unknown58
    Unknown59
    Unknown60
    Unknown61
    Unknown62
    Unknown63
    Unknown64
    Unknown65
    Unknown66
    Unknown67
    Unknown68
    Unknown69
    Unknown70
    Unknown71
    Unknown72
    Unknown73
    Unknown74
    Unknown75
    Unknown76
    Unknown77
    Unknown78
    Unknown79
    Unknown80
    Unknown81
    Unknown82
    Unknown83
    Unknown84
    Unknown85
    Unknown86
    Unknown87
    Unknown88
    Unknown89
    Unknown90
    Unknown91
    Unknown92
    Unknown93
    Unknown94
    Unknown95
    Unknown96
    Unknown97
    Unknown98
    Unknown99
    Unknown100
    Unknown101
    Unknown102
    Unknown103
    Unknown104
    Unknown105
    Unknown106
    Unknown107

- Supported formats (input only):
    CapMaker Plus
    Captionate
    Cavena 890
    Cheetah Caption
    CHK
    Matroska (.mkv)
    Matroska subtitle (.mks)
    NCI Caption
    Avid STL
    PAC (Screen Electronics)
    spt
    Ultech 1.30 Caption

- For Blu-ray .sup output use: 'Blu-raysup'
- For VobSub .sub output use: 'VobSub'
- For DOST/image .dost/image output use: 'Dost-image'
- For BDN/XML .xml/image output use: 'BDN-XML'
- For FCP/image .xml/image output use: 'FCP-image'
- For plain text only output use: 'Plaintext'
````
